src/main/kotlin/no/nav/klage/oppgave/service/unleash/ByEnhetStrategy.kt

```diff
@@ -37,7 +37,7 @@ class ByEnhetStrategy(val saksbehandlerRepository: SaksbehandlerRepository) : St
         }
 
     private fun getSaksbehandlersEnheter(unleashContext: UnleashContext) =
-        saksbehandlerRepository.getTilgangerForSaksbehandler(unleashContext.userId.get()).enheter
+        axsysClient.getTilgangerForSaksbehandler(unleashContext.userId.get()).enheter
```

Comment:
Hva er bakgrunnen til å gå rett mot `client` her?

Reply:
Jeg fikk sirkulære avhengigheter i Spring..

Reply:
Ups. Det høres ikke bra ut. Kanskje trengs noe opprydding blant "lagene"?

Reply:
Unleash-koden bruker Axsys for å hente ut enhetene saksbehandleren har tilgang til. Hvis jeg da lar unleash bruke Repoet, og så samtidig ønsker å featuretoggle noe i repoet, så blir det sirkulært. Når jeg bypasser repoet og lar Unleash bruker klienten direkte, så kunne jeg featuretoggle repoet. Jeg er sikker på at det kunne vært løst på andre måter også, så hvis du har noen tanker så kan du gjerne prøve å fikse det og lage en pr på det. Man kunne jo f.eks splittet opp repoet i to, men jeg følte ikke egentlig at det ville være noe bedre..

Reply:
Jeg merger dette nå, så kan vi evt ta dette punktet videre i en egen PR.